include/aarch64/vmlinux.h

```diff
@@ -1 +1 @@
-vmlinux_601.h
\ No newline at end of file
+vmlinux_6.6.h
```

Comment:
is it possible to maintain `vmlinux_606.h` naming? It has a tiny advantage of alphabetic sorting having meaningful version sorting behavior.

Reply:
I don't know what this zero in there is supposed to signify. Can you help me understand? In any event, I was going to remove all other headers next.

Reply:
that zero is there to allow for double-digit minor version, so that we have 6.6 -> 606, and 6.15 -> 615 (we don't believe we'll need triple digit minor versions, clearly).

we use this format internally, seems to work nicely, so unless there is a reason to not do that, why not keeping it?

Reply:
> so unless there is a reason to not do that, why not keeping it?

Because it's actively confusing. There is (or at least can be) version 6.0.9 which every person would intuitively associate with this string.

Reply:
As a counter example, tags in the Linux repository, where having a proper sorting order *actually* would make a lot of sense, don't adhere to this format. Why not? Well, presumably because it's more important to actually have a truthful representation of what is being referenced.

Reply:
sure, if you feel strongly